docs.json

```diff
@@ -183,26 +183,28 @@
             "pages": ["components/overview"]
           },
           {
-            "group": "Data Collection",
+            "group": "Data Collection (Wasm)",
             "pages": [
               {
                 "group": "Analytics",
                 "icon": "display-chart-up",
                 "pages": [
-                  "components/data-collection/amplitude",
                   "components/data-collection/google-analytics",
                   "components/data-collection/piano-analytics",
+                  "components/data-collection/amplitude",
```

Comment:
I understand we want to keep GA at the top, but these were intentionally sorted alphabetically (otherwise it looks a bit random)